oauth2-authorization-server/src/main/java/org/springframework/security/config/annotation/web/configurers/oauth2/server/authorization/OAuth2AuthorizationServerConfigurer.java

```diff
@@ -425,14 +424,4 @@ private void initEndpointMatchers(ProviderSettings providerSettings) {
 				"/.well-known/oauth-authorization-server", HttpMethod.GET.name());
 	}
 
-	private static void validateProviderSettings(ProviderSettings providerSettings) {
```

Comment:
This should not be removed. We could improve this validation by ensuring there is no `query` and `fragment` component.

Reply:
Okey